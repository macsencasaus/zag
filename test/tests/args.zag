// TEST: lex, ir

extern fn printf(fmt: *u8, ...) i32;

fn main(argc: i32, argv: **u8) i32 {
    printf(&"argc: %d, argv[0]: %s\n", argc, argv[0]);
    return 0;
}
