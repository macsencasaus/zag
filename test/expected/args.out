argc: 1, argv[0]: /tmp/zag_tmp
